pages/posts/3.tsx

```diff
@@ -0,0 +1,213 @@
+000001.sst	KEYREGISTRY	blocks		datastore	keystore	      version
+									`}</code>
+                </pre>
+                <p>
+                  Much of the content of these files is encrypted. They are only meant to be accessed through the Wesh API. We aren&apos;t going to look at all of these files or
+                  describe their format, but seeing them gives us a chance to discuss the basic types of information in Wesh.
+                </p>
+                <p>The most important to discuss are the three types of information associated with your identity:</p>
+
+                <p>
+                  <ul>
+                    <li>
+                      The <a href="https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md#peer-ids">libp2p host Peer ID</a> is what the example prints. It identifies your
+                      computer as a node in libp2p communication.
+                    </li>
+                    <li>
+                      The <a href="https://berty.tech/docs/protocol/#account-creation">Account key pair</a> holds the primary identity that you use to connect to other Wesh users and to
+                      join a group.
+                    </li>
+                    <li>
+                      A <a href="https://berty.tech/docs/protocol/#linking-devices">Device key pair</a> is created on each computer or mobile device where you use Wesh, and is connected
+                      cryptographically to your Account key pair.
+                    </li>
+                  </ul>
+                </p>
+
+                <p>
+                  In Wesh, communication is based on the “group” where messages are secure between group members. For each new group, the Wesh service generates a Metadata log
+                  (with info like members joining), a Message log (which we&apos;ll explore in a future blog post), and cryptographic secrets (your own secrets and secrets received from
+                  other group members). There are three types of groups:
+                </p>
+
+                <p>
+                  <ul>
+                    <li>
+                      Your <a href="https://berty.tech/docs/protocol/#account-group">Account group</a> stores all the secrets and metadata of your account and allows the devices linked
+                      to it to sync with each other, as explained above. So, when the example first calls <code>`NewPersistentServiceClient`</code>, Wesh creates one Account key pair, one
+                      Device key pair for your computer, and the Account group for it.
+                    </li>
+                    <li>
+                      When you <a href="https://berty.tech/docs/protocol/#adding-contacts">add another Wesh user as a contact</a>, you are both in a **[Contact
+                      group](https://berty.tech/docs/protocol/#contact-group)** where just the two of you can communicate securely. No other users can join this group.
```

Comment:
Need to make html: **[Contact
                      group](https://berty.tech/docs/protocol/#contact-group)**